### PR TITLE
Fix wrong value being passed to validation.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
This PR fixes an issue where validating how much space was available the wrong value was being passed to the validation, and thus it would fail in several instances. In particular it would fail for upload validation checking.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by awels** [comment]
ci test please

File: pkg/importer/dataStream.go [hunk 1]
```diff
@@ -853,7 +853,7 @@ func (d *DataStream) isHTTPQcow2() bool {
 }
 
 func (d *DataStream) calculateTargetSize(dest string) int64 {
-	targetQuantity := resource.NewScaledQuantity(util.GetAvailableSpace(filepath.Dir(dest)), 0)
+	targetQuantity := resource.NewScaledQuantity(d.AvailableSpace, 0)
```
Comment (mhenriks) [inline pkg/importer/dataStream.go]: Maybe this is something I missed in the scratch space PR.  But why is available space being cached?  My concerns are:
1.  One more thing to set when initializing a data stream
2.  If we forget to do above, the value will be zero, so nothing will work
3.  The name is ambiguous.  Free space where?

Comment (awels) [reply pkg/importer/dataStream.go]: The variable has been there for a while, its the space available in the target pvc. We added this variable when we didn't have scratch space and we calculated the availabe space after we wrote the file there already but before the resize, so we would not resize to the correct size. This is why we set it before doing anything (cached)

Comment (mhenriks) [reply pkg/importer/dataStream.go]: Maybe rename the variable in your datastream cleanup?

Comment (awels) [reply pkg/importer/dataStream.go]: I can rename it in this patch, as well, not a big deal not many places use it.

**Comment by awels** [comment]
@mhenriks I updated the name of the variable to reflects its purpose better.

**Pull request merged**